vspd.go

```diff
@@ -74,6 +82,31 @@ func run(ctx context.Context) error {
 	}
 	defer db.Close()
 
+	//Check if adminPass Hash is set in db.
+	hash, err := db.GetAdminHash()
+
+	//Ensure adminpass option is set
+	if cfg.AdminPass == "" && err != nil {
+		log.Error("Config error: the adminpass option is not set")
+		return err
+	}
+
+	if cfg.AdminPass != "" {
+		//Hash the cfg.AdminPass value
+		cfg.AdminPass, err = hashPassword(cfg.AdminPass)
+
+		if err != nil {
```

Comment:
No line space between error assignment and checking.
```suggestion
		cfg.AdminPass, err = hashPassword(cfg.AdminPass)
		if err != nil {
```